action/action-simulator/src/main/java/com/powsybl/action/simulator/tools/ActionSimulatorTool.java

```diff
@@ -63,8 +62,15 @@
 @AutoService(Tool.class)
 public class ActionSimulatorTool implements Tool {
 
+    private static final Supplier<ConversionOption> LOADER = Suppliers.memoize(() -> new DefaultConversionOption(CASE_FILE));
     private static final Logger LOGGER = LoggerFactory.getLogger(ActionSimulatorTool.class);
 
+    private final ConversionOption conversionOption;
+
+    public ActionSimulatorTool() {
+        this.conversionOption = LOADER.get();
```

Comment:
Is there an added value to using a supplier instead of plain constructor ?
(Same question for other tools)

Reply:
After some thinking, I think it does not add a lot of value so I deleted them.